.github/workflows/pythonpackage.yml

```diff
@@ -53,11 +53,12 @@ jobs:
         token: ${{ secrets.PAT_PULL_ENTERPRISE }}
         path: vaex-enterprise
     - name: install micromamba
-      uses: mamba-org/provision-with-micromamba@main
+      uses: mamba-org/setup-micromamba@v1
       with:
+        micromamba-version: 1.5.6-0
         environment-name: vaex-dev
         environment-file: ci/conda-env.yml
-        extra-specs: |
+        create-args: >-
```

Comment:
```suggestion
        timeout-minutes: 12
        create-args: >-
```

Reply:
Weird that this happened, I'll try an older version instead.